XMLProjectIS/Pages/Index.cshtml

```diff
@@ -5,43 +5,6 @@
     MovieDetails.Item[] movies = (MovieDetails.Item[])ViewData["MovieDetails"];
 }
 
```

Comment:
Deleted the styling classes from index and transferred them to site.css page file.

Reply:
Good.  In XHTML 1.0 Strict, inline and embedded styles are forbidden; linked is preferred.